api/VRDisplay.json

```diff
@@ -64,7 +77,7 @@
         },
         "status": {
           "experimental": true,
-          "standard_track": true,
+          "standard_track": false,
```

Comment:
@ddbeck I think this is no longer standards track, so changed `false` everywhere. OK?

Reply:
Yes, this is right. 👍